numalogic/udfs/trainer/_base.py

```diff
@@ -101,8 +103,13 @@ def compute(
             raise ConfigNotFoundError("Numalogic Trainer config not found!")
         dict_artifacts = {}
         trainer_cfg = numalogic_cfg.trainer
+        if trainer_transform:
+            input_ = trainer_transform.fit_transform(input_)
+            _LOGGER.info("Fit data using trainer transform")
```

Comment:
Might be too many logs here

Reply:
Oh this is just for testing the new changes. Let me remove it.